src/benchmarks/micro/libraries/System.Numerics.BitOperations/Perf_BitOperations.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace System.Numerics.Tests
 {
-    [BenchmarkCategory(Categories.Libraries, Categories.SIMD)]
+    [BenchmarkCategory(Categories.Libraries, Categories.SIMD, Categories.JIT)]
     public class Perf_BitOperations
```

Comment:
can you also tag Roslyn\CscBench.cs with "JIT"?

Reply:
Good catch, I forgot to save this file when I pushed the other changes.